lib/event-bus.js

```diff
@@ -16,11 +16,14 @@ module.exports = class EventBus {
         return;
       }
 
-      const evnt = JSON.parse(newValue);
+      const evnt = newValue && JSON.parse(newValue);
```

Comment:
Yeah, makes sense. Also, to solve my current problem, I might need to hack deeper. The code I'm currently testing navigates itself through multiple web pages, with its mocha tests getting written to stdout as multiple mocha reports. The current obstacle is aggregating the results into a single report. I thought about piping the stream to some existing library that can aggregate the results, or relaying these bus events 
to some parent process (eg, over ipc process.send) but I've run out of time for the project.
It looks like I'll need to set aside these tests, meh. I hope to come back to them, but am not sure when. So please feel free to do what you want with this PR-- leave it open, finish it yourself, or discard it entirely.